<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="17.769" tests="3" failures="1">
  <testsuite name="Root Suite" timestamp="2024-07-25T20:21:19" tests="0" file="cypress\e2e\login.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Funcionalidade Login" timestamp="2024-07-25T20:21:19" tests="3" time="17.755" failures="1">
    <testcase name="Funcionalidade Login Login com sucesso usando Comando customizado" time="5.018" classname="Login com sucesso usando Comando customizado">
    </testcase>
    <testcase name="Funcionalidade Login Login usando fixture" time="3.714" classname="Login usando fixture">
    </testcase>
    <testcase name="Funcionalidade Login Deve fazer login com sucesso - sem otimização" time="0.000" classname="Deve fazer login com sucesso - sem otimização">
      <failure message="Timed out retrying after 4000ms: expected &apos;&lt;p&gt;&apos; to contain &apos;Olá, Aluno&apos;" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: expected '<p>' to contain 'Olá, Aluno'
    at Context.eval (webpack:///./cypress/e2e/login.cy.js:35:65)]]></failure>
    </testcase>
  </testsuite>
</testsuites>